packages/node/src/utils.ts

```diff
@@ -73,9 +75,69 @@ export enum EdgeRuntimes {
   ExperimentalEdge = 'experimental-edge',
 }
 
+export enum NodejRuntime {
```

Comment:
Should this be plural to match the enum above?

```suggestion
export enum NodejRuntimes {
```

Reply:
<picture data-single-emoji=":facepalm:" title=":facepalm:"><img class="emoji" src="https://single-emoji.vercel.app/api/emoji/eyJhbGciOiJkaXIiLCJlbmMiOiJBMjU2R0NNIn0..4ZLnfzuDnh4G8acR.aOca7FtRTeDMQ51FWGkjDch4NFGb87-1nZtS7aoE14cmj7-UKbEjzqW8gKpG7QrykGt92h5-G6FKXoDRATNbzLNpzaPOjzeUPCIXELYTKHFJWekn0zHACg.x2Xc8y9f6k3-3gPqBP-4vw" alt=":facepalm:" width="20" height="auto" align="absmiddle"></picture> good catch!